docs/theme/overrides/home.html

```diff
@@ -191,6 +191,48 @@ <h2>Native performance</h2>
   </div>
 </section>
 
+<section class="mdx-parallax__group" data-md-color-scheme="slate" data-md-color-primary="indigo">
+  <div class="md-content md-grid" data-md-component="content">
+    <div class="md-content__inner mdx-installations md-typeset">
+      <header class="md-typeset">
+        <h1 id="powerhouse">
+          Automation Powerhouse
+          <a
+            href="#powerhouse"
+            class="headerlink"
+            title="Permanent link"
+          >
+            ¶
+          </a>
+        </h1>
+      </header>
+      <script>
+        fetch("https://api.github.com/repos/cirruslabs/tart/releases?per_page=100")
+          .then((response) => response.json())
+          .then((releases) => {
+            let allDownloads = 0;
+            for (let release of releases) {
+              for (let asset of release.assets) {
+                if (asset && asset.content_type === "application/octet-stream") {
+                  allDownloads += asset.download_count || 0
+                }
+              }
+            }
+            let counterElement = document.getElementById('installation-counter');
+            if (counterElement) {
+              counterElement.textContent = Math.round(allDownloads / 1000) + ",000"
+            }
+          })
+      </script>
+      <h2>
+        With <strong id="installation-counter">...</strong>  installations to date, Tart has been adopted for various scenarios.
```

Comment:
Some fallback in case of a HTTP request error.

```suggestion
        With more than <strong id="installation-counter">10,000</strong> installations to date, Tart has been adopted for various scenarios.
```

Reply:
🤣 I had this in the initial version! SGTM.